R/simulation.R

```diff
@@ -80,6 +80,15 @@ Simulation <- R6::R6Class(
     allFloatingMoleculeNames = function() {
       rClr::clrCall(private$.buildConfiguration, "AllPresentFloatingMoleculeNames")
     },
+    #' @description
+    #' Returns the mol weight value (in core unit) associated to the quantity with given path or NA if not found
+    #' @param quantityPath Path of quantity used to retrieve the molecular weight
+    molWeightFor = function(quantityPath) {

```

Comment:
@pchelle This is a method of a simulation. Please remove the code in the reporting engine that is trying to retrieve the molweight for a quantity path